pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Closes issue: <ISSUE>
+
+### This PR: 
+<!-- Describe what this PR adds to HotSHot -->
+<!-- E.g. -->
+<!-- * Implements feature 1 -->
+<!-- * Implements feature 2 -->
+<!-- * Fixes bug 3 -->
+
+### This PR does not: 
+<!-- Describe what is out of scope for this PR, if applicable -->
+<!-- * Implement feature 3 because that feature is blocked by Issue 4   -->
+
+### Key places to review: 
+<!-- Describe key places for reviewers to pay close attention to -->
+<!-- * file.rs, `add_integers` function -->
+
+### How to test this PR: 
+<!-- * E.g. `just async_std test` -->
+
+<!-- Complete the following items before creating this PR
+* Are the proper people tagged to review it?
+* Have you listed yourself as the assignee? 
```

Comment:
I'm wondering if this is necessary since the PR author is usually the same as the assignee. I think the author is also a filter attribute if we want to search for and filter PRs on GitHub.

Reply:
This was something @dailinsubjam brought up.  I am fine either way. :)

Reply:
I think this could help but not necessary, assignee's avatar will appear right after PR on the searching board which might be easier to find, but looks like we can also filter PR by selecting the author.

Reply:
Okay, I'll remove it!

Reply:
@dailinsubjam and @shenkeyao here is the new PR: https://github.com/EspressoSystems/HotShot/pull/2103

Reply:
Approved there. Thanks Ellie!